docs/guides/nwaku/configure-nwaku.md

```diff
@@ -0,0 +1,246 @@
+---
+title: Nwaku Configuration Examples
+---
+
+This guide provides detailed steps to configure a `nwaku` node for different use cases.
+
+## Connect to Other Peers
+
+To join the Waku Network, nodes must [bootstrap](/overview/reference/glossary#bootstrapping) for an entry point before discovering more peers. Nwaku provides multiple [peer discovery](/overview/concepts/peer-discovery) mechanisms:
+
+- [Configure Static Peers](/guides/nwaku/configure-discovery#configure-static-peers)
+- [Configure DNS Discovery](/guides/nwaku/configure-discovery#configure-dns-discovery)
+- [Configure Discv5](/guides/nwaku/configure-discovery#configure-discv5)
+- [Configure Peer Exchange](/guides/nwaku/configure-discovery#configure-peer-exchange)
+
+## Configure a Domain Name
+
+You can set up an IPv4 DNS domain name that resolves to the public IPv4 address of a node using the `dns4-domain-name` option. This allows the node's publicly announced multiaddrs to use the `/dns4` scheme.
+
+```bash
+./build/wakunode2 --dns4-domain-name=[DOMAIN NAME]
+```
+
+For example, consider the domain name `nwakunode.com`, which resolves to a `nwaku` node:
+
+```bash
+./build/wakunode2 --dns4-domain-name=nwakunode.com
+```
+
+Nodes with a domain name and secure WebSocket configured will generate a discoverable ENR with `/wss` multiaddr and `/dns4` domain name, essential for verifying domain certificates when connecting securely.
+
+:::info
+This example describes configuring a domain name that resolves to your node's IP address and is unrelated to [DNS Discovery](/overview/concepts/dns-discovery).
+:::
+
+## Configure Store Protocol and Message Store
+
+To enable the [Store protocol](/overview/concepts/protocols#store) in `nwaku`, use the following configuration options:
```

Comment:
This does not mean much to a casual reader (or even me for that matter).

Would suggest to rewrite as:

> to enable caching messages and serve them to other nodes in the network, enable the store protocol.